README.md

```diff
@@ -1,38 +1,15 @@
 # FunCoNN (BETA)
 
-Interactive map of nervous system wiring in the nematode _C. elegans_. Live at [FunCoNN.princeton.edu](https://funconn.princeton.edu/).
+Interactive map of functional connectivity measured by neural activtion in the nematode _C. elegans_. Live at [FunCoNN.princeton.edu](https://funconn.princeton.edu/).
 
-FunCoNN is built on top of [NemaNode](https://nemanode.org/) (NemaNode is reported in Witvliet et al., 2021) and adds Functional Connectivity data from the [Leifer Lab](http://leiferlab.princeton.edu) (citation pending) on top of the Chemical Synapse and Gap Junction data generated by the [Zhen](https://www.zhenlab.com), [Samuel](https://scholar.harvard.edu/aravisamuel), and [Lichtman](https://lichtmanlab.fas.harvard.edu) labs.
+FunCoNN is a fork of [NemaNode](https://nemanode.org/) (NemaNode is reported in Witvliet et al., 2021) that adds Functional Connectivity data from the [Leifer Lab](http://leiferlab.princeton.edu) on top of the Chemical Synapse and Gap Junction data generated by the [Zhen](https://www.zhenlab.com), [Samuel](https://scholar.harvard.edu/aravisamuel), and [Lichtman](https://lichtmanlab.fas.harvard.edu) labs.
```

Comment:
You might consider linking the word "fork" to their github repo since the word NemaNode is linked to their deployed instance.

Reply:
sure

Reply:
i believe i have now done that. if not, its also not soo important bercause we explicitly link to the original github repo a few paragraphs below.